scripts/update/Updater.php

```diff
@@ -507,6 +507,71 @@ public function update($initialVersion) {
         }
         $this->skip('3.6.6', '3.6.13');
 
+        if ($this->isVersion('3.6.13')) {
+
+            // create new column for test center and delivery
+            $persistenceId = $this->getServiceManager()->get(DeliveryMonitoringService::CONFIG_ID)->getOption(DeliveryMonitoringService::OPTION_PERSISTENCE);
```

Comment:
Update script can be costly, please provide a seperate script to migrate from one configuration to the other